pkgs/development/libraries/mlt/qt-5.nix

```
@@ -65,7 +72,7 @@ mkDerivation rec {
     # RPATH of binary /nix/store/.../bin/... contains a forbidden reference to /build/
     "-DCMAKE_SKIP_BUILD_RPATH=ON"
     "-DMOD_OPENCV=ON"
-  ];
+  ] ++ lib.optional enableGlaxnimate "-DMOD_GLAXNIMATE=ON"; # Enable glaxnimate https://github.com/NixOS/nixpkgs/issues/209923;
```

```suggestion
  ] ++ lib.optional enableGlaxnimate "-DMOD_GLAXNIMATE=ON";
```
that is clear from the line already and I am not sure why we link the issue here which can be closed after this is merged.